raml/api.raml

```diff
@@ -37,6 +37,19 @@ mediaType: application/json
             schema: !include schemas/200.json
             example: !include response_examples/200/regions_detailed.json
 
+  /regions/infra:
```

Comment:
I believe URL here should just be "/infra" (or "/api/v1/infra") to be homogenous with status, secutity issues, etc